src/test/ui/panic-runtime/abort-msvc-no-std.rs

```diff
@@ -0,0 +1,49 @@
+// build-pass
+// compile-flags: -C panic=abort
+// only-msvc
+// Test that `no_std` with `panic=abort` under MSVC toolchain
+// doesn't require `__CxxFrameHandler3`.
+
+#![no_std]
+#![no_main]
+
+use core::panic::PanicInfo;
+
+#[panic_handler]
+fn handle_panic(_: &PanicInfo) -> ! {
+    loop {}
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn memcpy(dest: *mut u8, _src: *const u8, _n: usize) -> *mut u8 {
+    dest
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn memmove(dest: *mut u8, _src: *const u8, _n: usize) -> *mut u8 {
+    dest
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn memset(mem: *mut u8, _val: i32, _n: usize) -> *mut u8 {
+    mem
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn memcmp(_mem1: *const u8, _mem2: *const u8, _n: usize) -> i32 {
+    0
+}
+
+#[no_mangle]
+#[used]
+static _fltused: i32 = 0;
+
+#[no_mangle]
+pub extern "C" fn mainCRTStartup() -> ! {
+    main()
```

Comment:
Can you add a call to a function that may panic, drops a value on panic and is not inlined from a crate that is compiled with `-Cpanic=unwind`? In that case the personality function should be referenced.

Reply:
A crate compiled with `panic=abort` can't import a crate compiled with `panic=unwind`.

Reply:
But what's the expected result? When both crates use `std`, it wouldn't be a problem, and the executable will run normally without any error; but when both use `no_std`, there's no unwind runtime. Should the executable pass or abort?

Reply:
The called function should not actually panic when running the executable. It should be added merely to check that there is no linking error when trying to call such a function.

Reply:
Ok, you're right. `__CxxFrameHandler3` is required in that case. Should I add back the empty implementation?

Reply:
I think so.